source/utilities.tex

```diff
@@ -14795,21 +14795,48 @@
 \end{ncbnf}
 
 \pnum
+Field widths are specified in defnadj{field width}{units};
+the number of column positions required to display a sequence of
+characters in a terminal.
+The \defnadj{minimum}{field width}
+is the number of field width units a replacement field minimally requires of
+the formatted sequence of characters produced for a format argument.
+The \defnadj{estimated}{field width} is the number of field width units
+that are required for the formatted sequence of characters
+produced for a format argument independent of
+the effects of the \fmtgrammarterm{width} option.
+The \defnadj{padding}{width} is the greater of \tcode{0} and
+the difference of the minimum field width and the estimated field width.
+
+\begin{note}
+The POSIX \tcode{wcswidth} function is an example of a function that,
+given a string, returns the number of column positions required by
+a terminal to display the string.
+\end{note}
+
+\pnum
+The \defnadj{fill}{character} is the character denoted by
+the \fmtgrammarterm{fill} option or,
+if the fill option is absent, the space character.
+For a format specification in a Unicode encoding,
+the fill character corresponds to a single UCS scalar value.
```

Comment:
Are the following changes, which the paper for #6126 made for existing wording, not required here?
```suggestion
For a format specification in UTF-8, UTF-16, or UTF-32,
the fill character corresponds to a single Unicode scalar value.
```

Reply:
Opened #6155 for this.

Reply:
Added Unicode fixes in a separate edit